karma.conf.maker.js

```diff
@@ -170,6 +170,15 @@ module.exports = function(codeCoverage, browserstack, watchMode, file) {
 
     plugins: plugins
   }
+
+  // add preprocessors when file is passed
+  if (file) {
+    config.files.forEach((file) => {
+      config.preprocessors[file] = ['webpack', 'sourcemap'];
+    });
+    delete config.preprocessors['test/test_index.js'];
+  }
```

Comment:
There is option to run only a single file for testing
```
gulp test-coverage --file=file.js
```

But its broken! idk if it was like that from beginning or in between. There were import statement errors, as files were not getting processed by webpack

sample command: `gulp test-coverage --file=test/spec/api_spec.js`
```
10 08 2020 21:48:09.629:INFO [HeadlessChrome 84.0.4147 (Windows 10.0.0)]: Connected on socket 4VcosmsxDu-BPFt6AAAA with id 82264559
HeadlessChrome 84.0.4147 (Windows 10.0.0) ERROR
  Uncaught ReferenceError: require is not defined
  at test/spec/api_spec.js:1:14
  
  ReferenceError: require is not defined
      at test/spec/api_spec.js:1:14
```


So I have fixed that here. Now its possible to run only a single file for testing

Reply:
Thank you @monis0395 for fixing the broken tool, i had issue with it last week.
Can you please mention in PR details that this PR fixes it, it will be good if we mention it in release notes too, good to inform all that it is fixed.